pkg/node/node.go

```diff
@@ -47,6 +47,14 @@ type Node struct {
 
 	// dev contains the device name to where the IPv6 traffic should be send
 	dev string
+
+	// IPv4HealthIP if set, this is the IPv4 address of the cilium-health
+	// endpoint located on the node.
+	IPv4HealthIP *net.IP
```

Comment:
`net.IP` is a slice, so it can already take a `nil` value. So do you really need to use a pointer here?